sqlitecluster/SQLiteNode.h

```diff
@@ -216,7 +216,11 @@ class SQLiteNode : public STCPManager {
     SQLitePeer* _getPeerByID(uint64_t id) const;
 
     // Look up the correct peer by the name it supplies in a NODE_LOGIN message.
-    SQLitePeer* _getPeerByName(const string& name) const;
+    SQLitePeer* _getPeerByName(const string& name);
+
+    // Search for then name in the peer list and return 0-based
+    // index if the name is found, -1 otherwise
```

Comment:
Typo?
```suggestion
    // Search for the name in the peer list and return 0-based
    // index if the name is found, -1 otherwise
```

Reply:
What is "0-based"?

Reply:
zero based like an array. makes sense?

Reply:
Hm, it makes sense when you explain it, but I think it is not a common expression. I say it like this: `returns the index if the name is found, -1 otherwise`